### ISSUE TITLE
crash when trying to open the crafting grid

### ISSUE THREAD

**Issue by Sailren** [raoulvdberge/refinedstorage#20]
http://paste.ubuntu.com/15603135/

used version is refinedstorage{0.5.1} [Refined Storage] (refinedstorage-0.5.1.jar)

**Comment by raoulvdberge** [comment]
What happens if you remove RebornCore / TechReborn? Does it still crash?

**Comment by Sailren** [comment]
same crash report, doesn't fix it

**Comment by raoulvdberge** [comment]
That is really weird, I'll investigate this.

**Comment by Sailren** [comment]
this is the current one:

http://paste.ubuntu.com/15604030/

**Comment by raoulvdberge** [comment]
`java.lang.IllegalAccessError: tried to access field net.minecraft.client.gui.inventory.GuiContainer.field_147003_i from class refinedstorage.gui.Scrollbar` it's trying to access a field `field_147003_i` which is guiLeft and is public:
```
/** Starting X position for the Gui. Inconsistent use for Gui backgrounds. */
public int guiLeft;
```
So this shouldn't throw an illegal access error... hmmm.

**Comment by raoulvdberge** [comment]
If you install the latest JEI, is the problem resolved? I need this for debugging sake :P

**Comment by Sailren** [comment]
yeah, that fixes it

**Comment by raoulvdberge** [comment]
Thanks.

**Comment by raoulvdberge** [comment]
Can you replace your current refined storage with this version, *and* remove JEI. Is it then fixed?

(change .zip to .jar)
[refinedstorage-0.5.2.zip](https://github.com/raoulvdberge/refinedstorage/files/201554/refinedstorage-0.5.2.zip)

**Comment by Sailren** [comment]
still the same crash with same data, i have to go to bed now, i hope u will find the error or some1 else, we really need something like AE1 in 1.9 :D

**Comment by raoulvdberge** [comment]
This is fixed in version 0.5.2, thanks for reporting!
